plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1086,7 +1078,7 @@ def main():
                 module.exit_json(changed=False, msg="VM %s does not exist" % vmid)
 
             if getattr(proxmox.nodes(vm[0]['node']), VZ_TYPE)(vmid).status.current.get()['status'] == 'running':
-                module.exit_json(changed=False, msg="VM %s is running. Stop it before deletion." % vmid)
+                module.fail_json(changed=False, msg="VM %s is running. Stop it before deletion." % vmid)
```

Comment:
This part was fixed by the introduction of a new meaning to the `force` parameter.